frappe/templates/web.html

```diff
@@ -4,14 +4,13 @@
 {% block content %}
 <div class="page-container" id="page-{{ name or page_name }}" data-path="{{ pathname }}"
 	{% if page_or_generator=="Generator" %}data-doctype="{{ doctype }}"{% endif %}>
-		<div class="page-content {% if not show_sidebar %}page-max-width{% endif %}">
 			<div class="row {% if show_sidebar %}vert-line{% endif %}">
 				{% if show_sidebar %}
 					<div class="col-sm-3 sidebar-block">
 						{% include "templates/includes/web_sidebar.html" %}
 					</div>
 				{% endif %}
-				<div class="{% if show_sidebar %}content-block col-sm-9{% else %} content-block col-sm-12 {% endif %}">
+				<div class="{% if show_sidebar %}page-content col-sm-9{% else %} page-content col-sm-12 {% endif %}">
```

Comment:
Okay! then it makes sense.

On Fri, May 6, 2016 at 3:30 PM, KanchanChauhan <notifications@github.com>
wrote:

> In frappe/templates/web.html
> <https://github.com/frappe/frappe/pull/1744#discussion_r62310345>:
>
> >  			<div class="row {% if show_sidebar %}vert-line{% endif %}">
> >  				{% if show_sidebar %}
> >  					<div class="col-sm-3 sidebar-block">
> >  						{% include "templates/includes/web_sidebar.html" %}
> >  					</div>
> >  				{% endif %}
> > -				<div class="{% if show_sidebar %}content-block col-sm-9{% else %} content-block col-sm-12 {% endif %}">
> > +				<div class="{% if show_sidebar %}page-content col-sm-9{% else %} page-content col-sm-12 {% endif %}">
>
> content-block was something I added, it was not there before.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/frappe/frappe/pull/1744/files/eeb7da7c05a93167c4d779972de8a8e8daba35d1#r62310345>
>